domain/main.go

```diff
@@ -226,3 +226,16 @@ func (e *employeeService) AddVacation(ctx context.Context, req *proto.VacationRe
 	}
 	return entities.VacationFromStorageToProto(vacation), nil
 }
+
+func (e *employeeService) Vacations(req *proto.GetAllRequest, stream proto.EmployeeService_VacationsServer) error {
+	vacations, err := e.repository.Vacations()
+	if err != nil {
+		return err
+	}
+	for _, vacation := range vacations {
+		if err := stream.Send(entities.VacationFromStorageToProto(vacation)); err != nil {
+			return err
```

Comment:
Think about wrapping errors where it makes sense, to improve debugging. https://blog.golang.org/go1.13-errors